### PR TITLE
fix word language model script and readme

### LINKED ISSUE #10224: Language model example cannot be run
The language model example cannot be run without manually creating a data folder.

https://github.com/apache/incubator-mxnet/tree/master/example/gluon/word_language_model

Also, the document is inconsistent with the code.  For example, optional argument 
**--data DATA        location of the data corpus**
does not appear in the code train.py

### PR DESCRIPTION
**Pull request by szha**
## Description ##
Fixes #10224 
@szhengac

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] ~~All changes have test coverage~~ manually tested on my machine
- [x] Code is well-documented
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] Fix left-over problems from previous updates in word language model example

### CODE REVIEW

**Comment by szha** [comment]
Because the model is a block, there will likely be no difference in performance if hybridize.

**Comment by szha** [comment]
An example where hybridize could make a difference is in image classification, which can be found in https://github.com/apache/incubator-mxnet/blob/master/example/gluon/image_classification.py#L254-L255

**Comment by szhengac** [comment]
but it may be useful if some internal blocks were hybridblock?

File: example/gluon/word_language_model/README.md [hunk 1]
```diff
@@ -1,67 +1,59 @@
...
+The wikitext-2 data is from [(The wikitext long term dependency language modeling dataset)](https://www.salesforce.com/products/einstein/ai-research/the-wikitext-dependency-language-modeling-dataset/). The training script automatically loads the dataset into `$PWD/data`.
 
 
 ## Usage
 
 Example runs and the results:
 
 ```
-python train.py -data ./data/ptb. --cuda --tied --nhid 650 --emsize 650 --dropout 0.5        # Test ppl of 75.3 in ptb
-python train.py -data ./data/ptb. --cuda --tied --nhid 1500 --emsize 1500 --dropout 0.65      # Test ppl of 72.0 in ptb
```
Comment (sxjscience) [inline example/gluon/word_language_model/README.md]: Suggest we should not remove the PTB experiments. I think it's originally included in the gluon example.

Comment (szha) [reply example/gluon/word_language_model/README.md]: We can bring it back once the CorpusReader is available in the main branch.

**Comment by szha** [comment]
It's mostly useful when many HybridBlocks are stacked together. Right now there are no such cases.

File: example/gluon/word_language_model/README.md [hunk 2]
```diff
@@ -1,67 +1,59 @@
 # Word-level language modeling RNN
 
-This example trains a multi-layer RNN (Elman, GRU, or LSTM) on Penn Treebank (PTB) language modeling benchmark.
+This example trains a multi-layer RNN (Elman, GRU, or LSTM) on WikiText-2 language modeling benchmark.
 
-The model obtains the state-of-the-art result on PTB using LSTM, getting a test perplexity of ~72.
-And ~97 ppl in WikiText-2, outperform than basic LSTM(99.3) and reach Variational LSTM(96.3).
+The model obtains ~97 ppl in WikiText-2 using LSTM, outperforming basic LSTM(99.3) and reach Variational LSTM(96.3).
```
Comment (szhengac) [inline example/gluon/word_language_model/README.md]: The current default setting is problematic. After running the experiment, the best testing ppl is 106.9

Comment (szha) [reply example/gluon/word_language_model/README.md]: Will update the number. Let's find a decent setting and put it back in later.

**Comment by sxjscience** [comment]
I’ll merge it first. We will submit another PR to fix the network structure.

**Pull request merged**